jenkins.check_copyright.sh

```diff
@@ -32,7 +32,7 @@ git diff --name-only `git merge-base origin/master HEAD` HEAD |
     grep -v -E '\.def$' |
     grep -v -E '\.inc$' |
     grep -v -E 'test/benchmarks/.*\.js$' |
-    xargs -I % sh -c "echo 'Check Copyright > Checking %'; python jenkins.check_copyright.py % > $ERRFILETEMP; if [ \$? -ne 0 ]; then cat $ERRFILETEMP >> $ERRFILE; fi"
+    xargs -I % sh -c "echo 'Check Copyright > Checking %'; python jenkins.check_copyright.py % > $ERRFILETEMP; || cat $ERRFILETEMP >> $ERRFILE; fi"
```

Comment:
@dilijev Got the following when trying `git push --force` :+1: 
`fatal: The current branch update-copyrightCheck has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin update-copyrightCheck
`
and the following error when I tried the `git push --set-upstream origin update-copyrightCheck` : 
`error: src refspec update-copyrighCheck does not match any.
error: failed to push some refs to 'https://github.com/Francososa/ChakraCore.git'
`
Any thoughts?